cts/cli/regression.tools.exp

```diff
@@ -8739,7 +8739,7 @@ cts-cli
 =#=#=#= End test: Get active shadow instance's file name (copied) (XML) - OK (0) =#=#=#=
 * Passed: crm_shadow     - Get active shadow instance's file name (copied) (XML)
 =#=#=#= Begin test: Get active shadow instance's contents (copied) =#=#=#=
-<cib epoch="1" num_updates="173" admin_epoch="1">
+<cib epoch="1" num_updates="173" admin_epoch="1"> update-origin="cluster01" update-client="crmd" update-user="hacluster" have-quorum="1" dc-uuid="2">
```

Comment:
?

Reply:
Same thing happened on main branch. As far as I can tell, some of the sanitization was more aggressive than it needed to be. We can cut this out if you want though.

Reply:
I don't see it on main. Just need to figure out why the `>` is in the wrong place and fix it

Reply:
https://github.com/ClusterLabs/pacemaker/pull/3642/commits/942c760b06f727091ef4596bb5f7c6f64ed1d94a#diff-6e0f7d808fa417f92613a0760eace3fc813b73adc89f4a228e4a2d320cb2c9e4R93

etc.

Reply:
Previously we were cutting out everything from `cib-last-written` through the end of the attributes. Now we cut out specific attributes (`cib-last-written`, generation attrs, `validate-with`)

Reply:
It looks good on main. The issue here is the `>` placement

Reply:
Ohhhh